rflx/error.py

```diff
@@ -73,6 +73,7 @@ class Subsystem(Enum):
     MODEL = auto()
     CLI = auto()
     GRAPH = auto()
+    SESSION = auto()
```

Comment:
Sessions are a part of the model. I would propose to remove `SESSION` and use `MODEL` instead. Or is there any advantage of differentiating different parts of the model?

Reply:
No, removed in 17d1f86.